EnglishFilter/sections/specific.txt

```diff
@@ -241,6 +241,7 @@ youtube.com#%#//scriptlet("json-prune", "adPlacements playerAds")
 !
 !
 ||javhoho.com/hzhxexywhxpy.php
+||walletinvestor.com/uploads/banners/
```

Comment:
We have a similar rule, so make a common rule like this `/uploads/banners/*domain=`